### PR TITLE
fixing a simple mistake in the doc

### PR DESCRIPTION
**Pull request by Chengxuan**
Updated $y==x^3 + x + e_0 + e_1$ to $y==x^3 + e_0x + e_1$ to match the circuit logic

### CODE REVIEW

File: src/usage/frontend-circom.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ In the following image, the `ivc_input`=$z_i$, the `external_inputs`=$w_i$, and
 
 <br>
 
-So for example, the following circuit proves (at each folding step) knowledge of $x$ such that $y==x^3 + x + e_0 + e_1$ for a known $y$ ($e_i$ are the `external_inputs[i]`):
+So for example, the following circuit proves (at each folding step) knowledge of $x$ such that $y==x^3 + e_0x + e_1$ for a known $y$ ($e_i$ are the `external_inputs[i]`):
```
Comment (arnaucube) [inline src/usage/frontend-circom.md:30]: should it be?
```suggestion
So for example, the following circuit proves (at each folding step) knowledge of $x$ such that $y==x^3 + e_0 + e_1$ for a known $y$ ($e_i$ are the `external_inputs[i]`):
```

**Review by arnaucube — COMMENTED** [review]
oh right! thanks for the fix

File: src/usage/frontend-circom.md (hunk 1 above)
Comment (Chengxuan) [reply src/usage/frontend-circom.md:30]: my reasoning was:
```
    temp1 <== ivc_input[0] * ivc_input[0];
    temp2 <== ivc_input[0] * external_inputs[0];
    ivc_output[0] <== temp1 * ivc_input[0] + temp2 + external_inputs[1];
```
temp1 = x^2
temp2 = e0*x

so => output = temp1 * x + temp2 + e1 =>  `x^3 + e0*x + e1`

Comment (arnaucube) [reply src/usage/frontend-circom.md:30]: oh right! my bad! I got confused, yes you're right. Thanks for the fix and sorry for the confusion!

**Review by arnaucube — APPROVED** [review]

**Pull request merged**